src/Conf.ml

```diff
@@ -111,7 +110,11 @@ module C : sig
 
   val int : default:int -> docv:string -> int option_decl
 
-  val get : 'a t -> 'a
+  val default : 'a t -> 'a
+
+  val update_by_name : config -> string * string -> config
```

Comment:
We should probably use the `update` function defined bellow and properly handle errors with some helpful error message.